kano_updater/return_codes.py

```diff
@@ -0,0 +1,89 @@
+# return_codes.py
+#
+# Copyright (C) 2018 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Return codes of binaries used throughout this project.
+
+
+from kano.logging import logger
+
+
+class RC(object):
+    """Return codes of binaries used throughout this project.
+
+    Rather than doing an ``exit`` with an RC from here, it is preferable to set
+    the RcState.rc. The ``main`` function should perform the exit with that.
+
+    Note:
+        ``kano-updater-recovery`` uses it's own RC class, see that source file.
+    """
+
+    SUCCESS = 0
+
+    NO_UPDATES_AVAILABLE = 10
+    CHECK_QUIET_PERIOD = 11
+
+    UNEXPECTED_ERROR = 20
+    WRONG_ARGUMENTS = 21  # TODO
+    INCORRECT_PERMISSIONS = 22  # TODO
+    MULTIPLE_INSTANCES = 23
+    NOT_PLUGGED_IN = 24
+
+    NO_NETWORK = 30
+    CANNOT_REACH_KANO = 31
+    HANGED_INDEFINITELY = 32  # TODO
+    SIG_TERM = 33
+    NOT_ENOUGH_SPACE = 34
+
+
+class RcState(object):
```

Comment:
This just looks a bit weird because it isn't `RCState`.

Reply:
Initially that looked weird but not so much now. Changed it everywhere